tools/oldest_constraints.txt

```diff
@@ -1,20 +1,20 @@
-# This file was generated by tools/pinning/oldest/repin.sh and can be updated using
+# This file was generated by /repin.sh and can be updated using
```

Comment:
this path got borked somehow. do you remember how you ran this script? the relevant code tries to handle this, but it clearly failed and it's written in bash so i shouldn't be surprised https://github.com/certbot/certbot/blob/0425b87b78413cc3e11159674d1a2e3d0610d7aa/tools/pinning/oldest/repin.sh#L11-L17

Reply:
i took a crack at this in https://github.com/certbot/certbot/pull/10128